docs/real-device-config.md

```diff
@@ -189,6 +188,28 @@ If this was successful, the output should end with something like:
     }
 ```
 
+#### Build for Generic iOS/tvOS devices, run manually
+
+As another way, you can build `WebDriverAgentRunner` for a generic iOS/tvOS device, and install the generated `.app` package to a real device.
+The generic one means for any iOS/tvOS devices similar to creating a release iOS/tvOS package.
+
+```bash
+# iOS
+$ xcodebuild clean build-for-testing -project WebDriverAgent.xcodeproj -derivedDataPath appium_wda -scheme WebDriverAgentRunner -target WebDriverAgentRunner -destination generic/platform=iOS CODE_SIGNING_ALLOWED=YES
+
+# tvOS
+$ xcodebuild clean build-for-testing -project WebDriverAgent.xcodeproj -derivedDataPath appium_wda -scheme WebDriverAgentRunner_tvOS -target WebDriverAgentRunner_tvOS -destination generic/platform=tvOS CODE_SIGNING_ALLOWED=YES
+```
+
+Then, `WebDriverAgentRunner-Runner.app` will be in `appium_wda/Build/Products/Debug-iphoneos/WebDriverAgentRunner-Runner.app` with the proper codesign by xcodebuild. Please make sure the `WebDriverAgent.xcodeproj` has proper condiguratin as this page to do sign properly.
```

Comment:
On successful completion the resulting package `WebDriverAgentRunner-Runner.app` should be located in `Build/Products/Debug-iphoneos/` subfolder under WebDriverAgent sources root.

Reply:
Please make sure the `WebDriverAgent.xcodeproj` has codesigning properties configured properly according to the above description if the build action fails.